### DISCUSSION TITLE
How to add multiple CPEs in CPE column for one component in dependency tracker

### DISCUSSION THREAD

**Discussion by shraddha0923** [DependencyTrack/dependency-track#2716] [General]
Hello, 

I would like to know is there any way to add multiple CPEs for one component in CPE field?
I do not want to create copy of component for different CPE ids. 

Please guide here.

**Comment by syalioune** [comment]
Hello @shraddha0923 
This is not possible. You'll have to create multiple components if you have different CPEs.
